e2e/test-cluster.yml.tpl

```
@@ -0,0 +1,43 @@
+---
+cluster_name: "{{ .ClusterName }}"
+template_name: "pavel-e2e-test"
+create_users: false
+grab_cluster_info: true
```

The files generated by the grab cluster info probably are not interesting in test deploys and consume a few min to grab them.

```suggestion
grab_cluster_info: false
```